__tests__/server.test.js

```diff
@@ -199,4 +199,59 @@ describe("backend API project", () => {
         });
     });
   });
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("that a POST request responds with the posted comment", () => {
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send({
+          username: "butter_bridge",
+          body: "Just posted my first comment !",
+        })
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.comment).toHaveProperty("author", "butter_bridge");
+          expect(body.comment).toHaveProperty(
+            "body",
+            "Just posted my first comment !"
+          );
+          expect(body.comment).toHaveProperty("votes", 0);
+          expect(body.comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(body.comment).toHaveProperty("article_id", 1);
+          expect(body.comment).toHaveProperty("created_at", expect.any(String));
+        });
+    });
+    test("that a POST request with a body that does not contain the correct fields returns 404 Bad request", () => {
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send({})
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad request");
+        });
+    });
+    test("that a POST request with a body of valid inputs but the value of a field is invalid returns 404 Bad request", () => {
+      return request(app)
+        .post("/api/articles/7/comments")
+        .send({
+          username: "sathice",
+          body: "are you throwing up furballs ?",
+        })
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad request");
+        });
+    });
+    test("that a POST request to a article that does not exist returns 404 Article not found", () => {
+      return request(app)
+        .post("/api/articles/61/comments")
+        .send({
+          username: "butter_bridge",
+          body: "Just posted my first comment !",
+        })
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Article not found");
+        });
+    });
+  });
```

Comment:
Nice error handling test, but you are missing an invalid article_id test I.e. /api/articles/banana/comments